actions/tag/create_tag.go

```diff
@@ -0,0 +1,43 @@
+package tag
+
+import (
+	"encoding/json"
+	"github.com/factly/data-portal-api/models"
+	"github.com/factly/data-portal-api/validation"
+	"github.com/go-playground/validator/v10"
+	"log"
+	"net/http"
+)
+
+// CreateTag - Create tag
+// @Summary Create tag
+// @Description Create tag
+// @Tags Tag
+// @ID add-tag
+// @Consume json
+// @Produce  json
+// @Param Tag body tag true "Tag object"
+// @Success 200 {object} models.Tag
+// @Failure 400 {array} string
+// @Router /tags [post]
+func createTag(w http.ResponseWriter, r *http.Request) {
```

Comment:
Exported function name should start with capital letter

Reply:
https://golang.org/ref/spec#Exported_identifiers

it is done to limit export scope from one package to another.